tests/README.md

```diff
@@ -1,76 +1,145 @@
-To run these tests, you need either of:
+# Tests
 
-1.    [Podman](https://github.com/containers/podman)
-1.    [Docker](https://github.com/docker)
+## Introduction
 
-The test-suite runs in a Podman/Docker container and supports two modes:
+This test-suite builds an OCI image matching the host OS with a fresh RPM
+installation layered on top (with `make install`), ready to be tested in a
+container, either automatically by a script (via `make check`) or in an
+interactive shell (via `make shell`).
 
-1. **Native** - Exercises the local build of RPM against a minimal image of the
-   Linux distribution running on the host.  Currently, only Fedora Linux is
-   supported.  This mode is optimized for local RPM development and requires
-   Podman.
+## Prerequisites
 
-1. **Standalone** - Exercises a custom, fresh build of RPM (including cmake
-   configuration) against a Fedora-based image.  This mode is optimized for
-   portability (CI environment) and works with both Podman and Docker.
+To use this test-suite, you need either of:
 
-The mode is selected automatically during cmake configuration based on the host
-distribution and the container engine installed, with native mode being
-preferred whenever possible, and is reported in the cmake output as follows:
+1. [Podman](https://github.com/containers/podman/)
+2. [Docker](https://github.com/docker/)
 
-    -- Using mktree backend: oci (native: <yes|no>)
+> [!IMPORTANT]
+> CMake integration (*native* mode) is currently only available on **Fedora
+> Linux** hosts with **Podman** installed, on other systems a fresh build of
+> RPM will be done as part of a Fedora-based image (*non-native* mode).
 
-Then run the command
+## Running all tests
+
+To build the image (if not built yet) and perform automated testing, run:
 
     make check
 
-The number of tests performed depends on features enabled at configure time,
-at least `--with-`/`--without-lua` and `--enable-`/`--disable-python`.
-See also the [INSTALL](../INSTALL) file for more information.
+The number of tests performed depends on features enabled at configure time, at
+least `--with-`/`--without-lua` and `--enable-`/`--disable-python`.  See also
```

Comment:
Lua is no longer an optional dependency so that can't affect the number of tests. Also, --enable/--disable and the like are autotools-era switches :smile:  The cmake counterpart for Python would be -DENABLE_PYTHON=ON/OFF.

Reply:
Heh indeed, fixed :smile: